_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: How to update to CIS AWS Foundations Benchmark v1.4.0
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update to version 1.4.0 of the CIS AWS Foundations Benchmark
+tags: ["aws", "security", "compliance"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-to-cis-14/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+
+This guide will walk you through the process of updating version 1.3.0 to version 1.4.0 of the CIS AWS Foundations Benchmark.
+If your infrastructure is already compliant with v1.3.0, and you are looking to upgrade to v1.4.0,
+then this guide is for you. Alternatively, if you are starting from scratch, check out our
+https://gruntwork.io/guides/compliance/how-to-achieve-cis-benchmark-compliance/[How to achieve compliance with the CIS AWS Foundations Benchmark]
+guide instead.
+
+=== Previous versions of this guide
+- https://gruntwork.io/guides/upgrades/how-to-update-to-cis-13/[How to update to CIS AWS version 1.3.0]
+
+=== What you'll learn in this guide
+
+This guide consists of two main sections:
+
+<<core_concepts>>::
+An overview of the CIS AWS Foundations Benchmark v1.4.0 and why it is important to update your code for compatibility.
+
+<<deployment_walkthrough>>::
+The steps you need to take to update your code to be compliant with v1.4.0. It includes a
+<<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+tag is compatible along with the manual steps you need to perform to achieve compliance with version v1.4.0 of the CIS
+AWS Foundations Benchmark.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
+
+
+==== Changes in recommendations
+Changes in recommendations (both additions and removals) are listed below. You can think of these as a "diff"
+between versions 1.3.0 and 1.4.0.
+
+===== New recommendations
+These are the new recommendations introduced in version 1.4.0 of the benchmark:
+
+- 2.1.3: Ensure MFA Delete is enabled on S3 buckets
+- 2.1.4: Ensure all data in Amazon S3 has been discovered, classified and secured when required
+- 2.3.1: Ensure that encryption is enabled for RDS Instances
+
+===== Updated recommendations
+Version 1.4.0 also updated a recommendation. It only affects the IAM password policy:
+
+- 1.12: Ensure credentials unused for 45 days or greater are disabled. The previous recommendation required credentials older than 90 days to be disabled. In addition, we also fixed a bug that was expiring all IAM passwords after 90 days regardless of whether they were used or not.
```

Comment:
+1, let's stick to explaining the diff between CIS 1.3 and 1.4 here.

Reply:
@ina-stoyanova @infraredgirl @marinalimeira okay let's remove this one. I don't think it's a critical bug fix, but I also wanted to avoid any unexpected surprises.

Reply:
I totally agree with you @robmorgan that we don't want unexpected surprises for our customers. I think in this case though as a new reader, I found it a bit out of context. 

I think we've all agreed so far it's safe to remove this, so I'll go ahead and commit the suggestion.